hw/ip/sram_ctrl/data/sram_ctrl.hjson

```diff
@@ -79,6 +79,20 @@
       type:      "int",
       default:   "0x1000"
     },
+    { name:      "InstSize",
+      desc:      "Memory size of a single RAM tile (in bytes).",
+      type:      "int",
+      default:   "4096"
+      local:   "false"
+      expose:  "true"
+    },
+    { name:      "NumRamInst",
```

Comment:
Why using this parameter if it can be derived from others? Seems redundant.

Reply:
Technically no need for that. Unfortunately, the topgen tooling cannot yet deal with the computed number of RAM instances. I'll file an issue on that, for a later improvement.

Reply:
Could this just be a localparam in the individual module holding the RAMs?

Reply:
Unfortunately no. This parameter is used as a dimension for the DFT signals, which depends on the number of RAM instances.